Test/MagazzinoTest.java

```diff
@@ -0,0 +1,62 @@
+import Management.Magazzino;
+import Products.ProdottoElettronico;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class MagazzinoTest {
+    private Magazzino magazzino;
+    private ProdottoElettronico prodotto1;
+    private ProdottoElettronico prodotto2;
+
+   @BeforeEach
+    public void setup() {
+        magazzino = new Magazzino();
+        magazzino.getMagazzino().clear();
```

Comment:
Perchè questo clear? In teoria un magazzino appena creato dovrebbe già essere vuoto.

Reply:
Effettivamente è inutile, lo rimuovo